formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -7,11 +8,11 @@
  * This has a nice side effect when printing the terms:
  * If a node has attributes attached to it,
  */
+
```

Comment:
Yeah, I was thinking the following use case:
```
/** The following family of functions does bar */

/** func1 does blah */
let func1 =  ...

/** func2 does blah */
let func2 = ...

/** func 3 does blah */
let func3 = ...

/** The following family of functions do foo */

/** func1 does blah */
let func1 =  ...

/** func2 does blah */
let func2 = ...

/** func 3 does blah */
let func3 = ...
```

Without the space between two doc comments, it would be formatted as : 
```
/** The following family of functions do bar */
/** func1 does blah */
let func1 =  ...

/** func2 does blah */
let func2 = ...

/** func 3 does blah */
let func3 = ...

/** The following family of functions does foo */
/** func1 does blah */
let func1 =  ...

/** func2 does blah */
let func2 = ...

/** func 3 does blah */
let func3 = ...
```